### PR TITLE
Separate handling mmbiblio from metamath-exe

### PR DESCRIPTION
**Pull request by david-a-wheeler**
Downloading and building metamath-exe should be *separate* from handling mmbiblio. Let's separate them.

### CODE REVIEW

File: scripts/build-metamath [hunk 1]
```diff
@@ -41,8 +41,8 @@ mkdir -p "$my_bindir"
 cp -p src/metamath "${my_bindir}"
 
 # If there's no more recent mmbiblio.html, use this one.
-if ! [ -e ../mmbiblio.html ] ; then
-  cp -p mmbiblio.html ../
-fi
+# if ! [ -e ../mmbiblio.html ] ; then
+#   cp -p mmbiblio.html ../
+# fi
```
Comment (benjub) [inline scripts/build-metamath:46]: If it's not needed, then remove it instead of commenting it out.

Comment (jkingdon) [reply scripts/build-metamath:46]: I'd lean towards removing it too, but don't consider this a blocker and am quite willing to defer to the choice that @david-a-wheeler will find more convenient in maintaining these scripts.

**Review by jkingdon — APPROVED** [review]
I won't claim to fully understand how this interacts with all the other scripts, but the idea of fetching mmbiblio.html individually from the live site seems..... odd. So this change, which gets rid of the odd thing, is good.

**Review by tirix — APPROVED** [review]
@david-a-wheeler you control the site generation, so you can choose the best time to merge this PR.

**Comment by jkingdon** [comment]
@david-a-wheeler Can this be merged now? Don't want to break any of the site generation scripts, but if it can be merged, let's do so.

**Comment by benjub** [comment]
@david-a-wheeler Can this be merged now?  (I would still remove the code you commented out, but only if it does not delay merging.)

**Comment by jkingdon** [comment]
I had forgotten about this pull request when I made #3462 but that's what the merge conflicts are about.

**Comment by benjub** [comment]
> I had forgotten about this pull request when I made #3462 but that's what the merge conflicts are about.

So, what should be done ? Close this MR, or solve conflicts and merge it ?

**Comment by jkingdon** [comment]
> > I had forgotten about this pull request when I made #3462 but that's what the merge conflicts are about.
> 
> So, what should be done ? Close this MR, or solve conflicts and merge it ?

OK with me if we close it.

If anyone wants to take it back up again, they can always re-open it or make a new one.

**Comment by tirix** [comment]
I solved the merge conflit, so both options are open.

**Comment by benjub** [comment]
Thanks. I would tend towards merging, but I may have missed some possible consequences. Up to you or @jkingdon.

**Comment by jkingdon** [comment]
> Thanks. I would tend towards merging, but I may have missed some possible consequences. Up to you or @jkingdon.

I guess I don't understand https://github.com/metamath/metamath-website-scripts well enough to even guess at consequences or lack of consequences.

**Comment by tirix** [comment]
I believe this PR has been superseded by #3593 and https://github.com/metamath/metamath-website-scripts/pull/16, so it should be closed.

After I resolved the conflicts, no change is left so closing or merging will have the same effect.

**Comment by jkingdon** [comment]
> I believe this PR has been superseded by #3593 and [metamath/metamath-website-scripts#16](https://github.com/metamath/metamath-website-scripts/pull/16), so it should be closed.
> 
> After I resolved the conflicts, no change is left so closing or merging will have the same effect.

Agreed, don't think this needs any elaborate discussion to close but if anyone sees differently feel free to reopen or add a comment.

**Pull request closed without merging**